components/automate-gateway/api/compliance/reporting/reporting.proto

```diff
@@ -323,20 +339,29 @@ message Statistics {
 }
 
 message SuggestionRequest {
+	// The type of resource to get suggestions for
 	string type = 1;
+	// The term to use to match resources on 
 	string text = 2;
+	// The maximum number of suggestions to return
 	int32 size = 3;
+	// The criteria used to filter the suggestions returned.
```

Comment:
This ends with a `.` to turn it from a `title` to a `description`. For some reason, the `title` of an array will not appear in the ReDoc UI.